### ISSUE TITLE
Accessibility:List: The “has details” information (Arrow-icons) about list items is not read, when list items are accessed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3481]
Description: The “has details” information (Arrow-icons) about list items is not read, when list items are accessed. 

Expected: The information such as “has details” should be conveyed to the user on accessing the list items.

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/94648466-38121000-0310-11eb-8a9d-59b219f9d642.png)
